local-volume/onetimecontroller/controller.go

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
A more clear name could be: local-volume/utils/update_pv_to_beta.go.

Can you also add a Dockerfile and sample deployment spec to run this as a Kubernetes job?

Reply:
sure, will do